libr/asm/p/asm_x86_nz.c

```diff
@@ -5158,10 +5159,11 @@ static int assemble(RAsm *a, RAsmOp *ao, const char *str) {
 		if (!r_str_casecmp (instr.mnemonic, lt_ptr->mnemonic)) {
 			if (lt_ptr->opcode > 0) {
 				if (!lt_ptr->only_x32 || a->bits != 64) {
-					ut8 *ptr = (ut8 *)&lt_ptr->opcode;
-					int i = 0;
-					for (; i < lt_ptr->size; i++) {
-						data[i] = ptr[lt_ptr->size - (i + 1)];
+					ut64 opcode = lt_ptr->opcode;
+					int i = lt_ptr->size - 1;
```

Comment:
Sizet

Reply:
https://github.com/radareorg/radare2/blob/c83d0e0c5f96aa8b01ca85b8df035c59452ed78d/libr/asm/p/asm_x86_nz.c#L4204-L4210
opcode is defined ut64. size_t may cause problem on arm32, windows...

Reply:
The comment is in line 5163, not 5163. I said nothing to the opcode

Reply:
lookuptable’s size field is int, why let i be size_t?